graalvm/core/src/test/java/org/infinispan/client/hotrod/graalvm/core/EmbeddedTest.java

```diff
@@ -1,4 +1,4 @@
-package org.infinispan.integrationtests.graalvm.core;
+package org.infinispan.client.hotrod.graalvm.core;
```

Comment:
is this change intentional?

Reply:
No, good spot. It seems I also change the CDI integration tests as well. I have reverted these changes, but I have updated the EmbeddedTest package to match those found in `src/main`.